office-backend/src/app.ts

```diff
@@ -29,7 +37,7 @@ fastify.addHook("onRequest", (request, reply, done) => {
       "Access-Control-Allow-Methods",
       "GET, POST, PUT, DELETE, OPTIONS",
     );
-    reply.header("Access-Control-Allow-Headers", "Content-Type, Authorization");
+    reply.header("Access-Control-Allow-Headers", "Content-Type, Authorization, auteur");
```

Comment:
`auteur`? What a strange term? Where does this variable come from? I don't see it in the code above?